prime-dotnet-webapi/Models/MongoModels/Locations.cs

```diff
@@ -0,0 +1,20 @@
+using Mongo.Migration.Documents;
+using Mongo.Migration.Documents.Attributes;
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+
+namespace Prime.Models
+{
+    [StartUpVersion("0.0.2")]
+    [CollectionLocation("Locations", "prime")]
+    public class Locations : IDocument
```

Comment:
We have a site model called "location" now, so this naming is unfortunate. Is it possible to re-name this model?
Could we also change the namespace to something to make clear this is a POC model?